pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/crystal/globalCrystal.css

```diff
@@ -578,7 +578,11 @@ For IE 9 and IE10, need to add padding to the bottom of some dialogs generated w
 }
 
 .pentaho-dialog .dialog-content {
-  padding: 5px 10px 10px 10px;
+  padding: 5px 0px 10px 0px;
+}
+
+.pentaho-dialog .dialog-content .pentaho-padding-lg {
+  padding: 6px;
```

Comment:
You're changing a very generic class. Are you sure this does not have an impact on other areas of PUC ?